tests/e2e/addons/TA_cim_addon/default/eventtypes.conf

```diff
@@ -0,0 +1,11 @@
+##
+## SPDX-FileCopyrightText: 2021 Splunk, Inc. <sales@splunk.com>
+## SPDX-License-Identifier: LicenseRef-Splunk-8-2021
+##
+##
+
+; [test_auth]
+; search = sourcetype=test:data:1 AND action IN ("success","failure","error")
```

Comment:
Let's delete it if it is commented anyways.

Reply:
Same for tags.conf.

Reply:
Done